xlml/utils/metric.py

```diff
@@ -251,6 +267,34 @@ def process_tensorboard_summary(
   return [metric_history_rows], [metadata_history_rows]
 
 
+def get_gcs_file_location_with_regex(file_location: str):
```

Comment:
Can we add unit tests for this func? Thanks!

Reply:
Done!